.pre-commit-config.yaml

```diff
@@ -55,7 +55,7 @@ repos:
             pyarrow>=1.0.1,
             chardet>=3.0.4,
             fastavro>=1.0.0.post1,
-            python-snappy>=0.5.4,
```

Comment:
why remove here? it is kept in `requirements.txt` file

Reply:
synced with requirements

Reply:
awesome